tool/resources/org/antlr/v4/tool/templates/codegen/Java/Java.stg

```diff
@@ -647,8 +647,9 @@ setState(<m.stateNumber>);
 <capture>
 if ( <if(invert)><m.varName> \<= 0 || <else>!<endif>(<expr>) ) {
 	<if(m.labels)><m.labels:{l | <labelref(l)> = (Token)}><endif>_errHandler.recoverInline(this);
+} else {
+  consume();
```

Comment:
Can you change this to a hard tab? You would not believe the file size difference we found by using hard tabs exclusively in the generated code.